anthos-bm-apigee/README.md

```diff
@@ -0,0 +1,81 @@
+
+- Enable Compute Engine API Services and Apigee API Services.
+
+- Ensure that the default Compute Engine developer account has Editor role. You can also create a new service account for the compute engine with Editor and specify the service account name in terraform variables.
+
+![Default Service Account](docs/images/default_service_account.png)
+
+- A [Service Account](https://cloud.google.com/iam/docs/creating-managing-service-accounts) in the project that satisfies **one** of the following requirements and its **[key file downloaded](docs/create_sa_key.md)** to the workstation:
+    - The Service Account has `Owner` and `Apigee Organization Administrator` permissions
+    - The Service Account has both `Editor`, `Project IAM Admin` and `Apigee Organization Administrator` permissions
+The scripts to create service account and key creation is also mentioned in Quickstart module.
+- Organizational Policy Constraints
+      Follwowing list of Policy needs to be enabled for the organizations. If these are not enabled at Organizations, please consider them for the project.
+
+      |  Policy Name                                 | Constraint Name                                   | Effective Polciy |
+      |  ------------------------------------------- | ------------------------------------------------- | ---------------- |
+      | Disable service account creation             | constraints/iam.disableServiceAccountCreation	   | Not Enforced     |
+      | Disable service account key creation         | constraints/iam.disableServiceAccountKeyCreation	 | Not enforced     |
+      | Restrict VM IP Forwarding                    | constraints/compute.vmCanIpForward	         | Allowed All      |
+      | Define allowed external IPs for VM instances | constraints/compute.vmExternalIpAccess	           | Allowed All      |
+      | Shielded VMs                                 | constraints/compute.requireShieldedVm	           | Not Enforced     |
+      | Require OS Login                             | constraints/compute.requireOsLogin	               | Not Enforced     |
+      | Skip default network creation                | constraints/compute.skipDefaultNetworkCreation	   | Not Enforced     |
+
+- Default Network with default Firewall policies
+The installation requires a network with name as default. If default network creation is enabled for the organization, the project will get them inherited. In case the  Skip default network creation is Enforced, you can create a new VPC network with name default in auto mode.
+![Default Network](docs/images/default_network.png)
+
+- Quota Check
+The demo Apigee instance requires 4 VMs with n1-standard-8 machine type. Please ensure there are enough quota set for CPU,Memory, IP Addresses for the region you are hosting the project.
+
+- **Sample Prerequisite script is also provided unnder resources folder as ./resources/run_prerequisite.sh**
```

Comment:
Can we add this as a note and also link to the exact file?

> **Note:** The repository also has a [utility script](./resources/run_prerequisite.sh) that checks and configures some of these pre-requisites.

Reply:
This is changed.